src/app/config/projectProperties.json

```diff
@@ -4,39 +4,54 @@
             "displayname": "Office Add-in project using Angular framework",
             "templates": {
                 "typescript": {
-                    "repository": ""},
+                     "repository": "",
+                    "branch": "master"},
```

Comment:
Instead of having to include "branch": "master" for everything, it could use "master" as the default if not specified, and only require "branch" when specifying something else. The code should be able to handle "branch" not being listed in the json object.